src/event-stream/reader.ts

```diff
@@ -121,8 +203,16 @@ export function parseMessageTransactions(msg: CoreNodeBlockMessage): CoreNodeMes
           throw new Error(`Could not find txid for process BTC tx: ${JSON.stringify(msg)}`);
```

Comment:
I think this shouldn't throw -- bitcoin operations could _fail_, but still be processed and result in a tx receipt (e.g., a stack-stx op that doesn't stack enough, or the caller doesn't have enough unlocked STX) -- these txs just wouldn't have events, though. I think the thing to do here would be to log some information and continue processing.

Reply:
Fixed. An empty/failed btc transaction like this can't currently be fit into the current db schema or API schemas, so they are simply ignored.